src/tactic/command/pipeline_task_trigger.py

```diff
@@ -373,27 +373,42 @@ def execute(my):
 
 
 class PipelineTaskCreateTrigger(Trigger):
+    
     def execute(my):
         input = my.get_input()
 
         search_key = input.get("search_key")
         task = Search.get_by_search_key(search_key)
-
-        parent = task.get_parent()
-
+        
         # get the definition of the trigger
         trigger_sobj = my.get_trigger_sobj()
         data = trigger_sobj.get_value("data")
         data = jsonloads(data)
```

Comment:
There should be try except for jsonloads.. in case it has some syntax error due to manual user edit.